src/components/AdditionColumn/AdditionColumn.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { act, fireEvent, render, screen } from "@testing-library/react";
+import { TestRendererWithContext } from "../../utils/testRendererWithContext";
+
+import AdditionColumn from "./AdditionColumn";
+
+import "jest-styled-components";
+
+beforeEach(() => {
+  act(() => {
+    render(<AdditionColumn />, {
+      wrapper: TestRendererWithContext,
+    });
+  });
+});
+
+describe("<AdditionColumn />", () => {
+  it("Default render - It should render closed state of the component", () => {
+    const closedColumnText = "Add another list";
+
+    expect(screen.getByText(closedColumnText)).toBeTruthy();
+  });
+
+  it("Press on the column - It should expand the component and show more items", async () => {
+    const closedColumnText = "Add another list";
+    const addButtonText = "Add card";
+    const editableTitlePlacehold = "Enter list title...";
+
+    // before press on the component - it should show default text
+    expect(screen.getByText(closedColumnText)).toBeTruthy();
+    expect(screen.getByTestId("AdditionColumn_expandableContainer")).toHaveStyleRule("opacity: 0");
```

Comment:
if you wanted to check component's style, use this `toHaveStyleRule` function. In order to do so, make sure to import `"jest-styled-components"`